### PR TITLE
Stack Tracing

### LINKED ISSUE #103: Add Stack Trace to Panic
It doesn't have to be fancy, but it would be nice to have.

### PR DESCRIPTION
**Pull request by Kfeavel**
Also fixed makefile issues related to panic screen
Closes #103

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,9 +12,9 @@
 
 SYSROOT	= sysroot
 INCLUDE = $(SYSROOT)/usr/include
-CPP_SRC = $(shell find kernel/ -name "*.cpp")
-ATT_SRC = $(shell find kernel/ -name "*.s")
-HEADERS = $(shell find $(INCLUDE) -name "*.hpp" -name "*.h")
+CPP_SRC = $(shell find kernel -name "*.cpp")
+ATT_SRC = $(shell find kernel -name "*.s")
+HEADERS = $(shell find $(INCLUDE) -name "*.hpp")
```
Comment (micahswitzer) [inline Makefile]: Do these lines need to be included in this PR?

Comment (Kfeavel) [reply Makefile]: Technically no. I should have changed that on dev.

Comment (micahswitzer) [reply Makefile]: Are you going to change this?

Comment (Kfeavel) [reply Makefile]: Ah, I forgot. I can change it back.

Comment (Kfeavel) [reply Makefile]: Pushed.

**Pull request merged**